Gemfile.lock

```diff
@@ -447,7 +447,7 @@ GEM
     mime-types-data (3.2020.1104)
     mimemagic (0.3.5)
     mini_histogram (0.3.1)
-    mini_magick (4.10.1)
+    mini_magick (4.11.0)
```

Comment:
dependency of `image_processing` which is used by `carrierwave`, fixes and new features: https://github.com/minimagick/minimagick/releases/tag/v4.11.0